### PR TITLE
Use sendgrid in production for SLO-reporter

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## Related Issues and Dependencies

Related-To: https://github.com/thoth-station/thoth-application/issues/863
Related-To: https://github.com/thoth-station/slo-reporter/issues/200

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

- Bump version of SLO reporter in prod to v0.9.0
- use send grid to send emails from cnv-prod cluster

### CODE REVIEW

File: slo-reporter/overlays/cnv-prod/secrets.enc.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ items:
...
...mGiVyBgpqQtIwqcuRInhIAzqCN+JfNHl7vV9m4SfCTGhhMk4NV4BqFbN/t/ADNc2JEAb9uLoYN5bp4dQ+QkeN691P6SJjUo+ZudhY8W7nJZiO7AmILFtmhEK8LsE1RmYQEUM/wckXmZBK3UWRUzGTtur5M0C26zG2fZD5iigay5hNX+veIP4XyDoI/cISvD/zMWfSybx9MRuEG4kCk1Lk+MfTQXRWYrHtnHJLCDX0JIzsCw5s3MUZnGiubrKrLTLwBaDmO3GeTeXEpmvK1qvCcABo2qN5pmDxRTjxXXaQwYU+h/um3APZgJZqluJU6xwN68NQcI/LHAtzZJdgIhnnaNfaRFXPuBi3b1hZToca8SyuTsoP0FeRfZEz9GviBXHHfeurNKDu82c+/SmcrE2p4titYST5cHdLrX55ZXX/hSt1WY075pIzeRpJ2jEpkse7tRyM+3Nboioilte+X4EwzUa1f7+2IDq3Mwq+bjqnWhO2WNn/5vb0+a4VAgMNBEn/MM0R/bjcT2QFeFnSJuSZKH6Ou/3zsPShXurMRtAPE6rdx98NPahYNv0NlFuBMVtUtzVUCGRf4wzdxE2I8wZ4KBDSveWWndslDQtLKCA12ysTjPeSs9vqckgYtiFpv06Sp4F2W+vokfsxHuZh+w4x/sDHFvmuog/4Ajiq4H7L7fL/6IhdYYWT64jz6oVfmxvE8LqaES//IvyJa3qCZ2Z2h6kJdo0VJCEcLRYdQ1m7/md4veNH+ie+z6gn8Kl/0lxZF6YtCSn9QQD8r6M/F7EukW3QAsfsLKC9XM8KIUCyxGp73WaBTfWiva9L+OUl2jlWgpRGLEZjpw5+YW23bp/56Tj4yqxdNBAIY4bwAM3CFGrJviX6qsHFYeHnTY0IWyCBUVVjjZ3TsQdAcwXZiDtG3jpRxoe91F5cKi/WwyG424nI0IIhwA4kQ689Z3b+p9850z6FtXwPfGMufiNck6CxvjQrW6OY6HG4yroVz96xfBvwFGUVSRVpd/78Ao8T9wVulTGKYqGTIxSiUDIEzqhYo82H3VJOP1do6lksrC3Biy3D/Tug/0OiR/kegkEBO5tKOZv8GiPZlX2taSFayo2l7KE=,iv:cR2qa7USMltmXDfKQiqGA6yNOPbtlRL31fVF+ioXnkE=,tag:uhaH2peeGv1F7VlUU/D7FA==,type:str]
```
Comment (harshad16) [inline slo-reporter/overlays/cnv-prod/secrets.enc.yaml:5]: is this token required ? wanted to know as the content of this might of the internal thanos.

**Review by harshad16 — COMMENTED** [review]
lgtm , thanks for the addition.
i would like to hold it.  will include this along with prod release.

/hold

File: slo-reporter/overlays/cnv-prod/secrets.enc.yaml (hunk 1 above)
Comment (pacospace) [reply slo-reporter/overlays/cnv-prod/secrets.enc.yaml:5]: It is not required in prod, but we should keep the empty variable in configmap or SLO-reporter will fail

Comment (harshad16) [reply slo-reporter/overlays/cnv-prod/secrets.enc.yaml:5]: ack, so what will happen if the same token stay will it break it ? or as you suggested we should change it with empty value, if so can please make that change in this pr.

Comment (pacospace) [reply slo-reporter/overlays/cnv-prod/secrets.enc.yaml]: I changed it to empty :) Thanks @harshad16 !

**Review by harshad16 — APPROVED** [review]
/unhold
thanks for this update.

**Pull request merged**